src/explore-education-statistics-admin/src/components/editable/EditableSectionBlocks.tsx

```diff
@@ -59,32 +55,33 @@ const EditableSectionBlocks = <T extends EditableBlock = EditableBlock>({
     return <InsetText>There is no content for this section.</InsetText>;
   }
 
+  if (isReordering) {
+    return (
+      <ReorderableList
+        id="reorder-sections"
+        list={blocks.map(block => ({
+          id: block.id,
+          label: <div className={styles.draggable}>{renderBlock(block)}</div>,
```

Comment:
Works great!  
One UX point: As the only way to add a text block in-between previous blocks is to add a new one at the end of the list and then reorder it into position, I wonder if we can improve the visual look of the draggable and empty text block, here it is quite slim there isn't much indication as to what it represents.
![image](https://github.com/user-attachments/assets/326a456d-dc9f-48bc-a3b6-5cde9bbd561e)
It would be great if we could alter the look of the empty "preview" text block to just make it a little more clear, like we do for the empty editable text block
![image](https://github.com/user-attachments/assets/5a7df306-0c76-4f83-b8ba-00cc49b79a19)

Reply:
Good spot, I've changed it to add a min height and the 'this section is empty' text.

![Screenshot 2024-11-14 153909](https://github.com/user-attachments/assets/3747a460-e0a0-423f-b03a-cfde6587ab35)